recipes/default.rb

```diff
@@ -21,6 +21,11 @@
 include_recipe 'hadoop::_hadoop_checkconfig'
 include_recipe 'hadoop::_compression_libs'
 
+# Load helpers
+Chef::Recipe.send(:include, Hadoop::Helpers)
+Chef::Resource::Link.send(:include, Hadoop::Helpers)
+Chef::Resource::Template.send(:include, Hadoop::Helpers)
```

Comment:
another suggestion possibly for a subsequent refactor: you can just include it into the Chef::Resource.  But out of concern for naming conflicts, it would be better to also convert the helper module methods to class methods (ie, ``self.hdp22?``), and then refer to them everywhere as ``Hadoop::Helpers.hdp22?``